scripts/ci/get-node-version.mjs

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env node
+import { getPackageJSON } from './_common.mjs'
+
+const DOCKERHUB_BASE_URL =
+  'https://hub.docker.com/v2/repositories/library/node/tags?page_size=100'
+
+const nodeVersion = await getPackageVersion()
+const version = await getVersion(nodeVersion)
+
+if (!version) {
+  process.stderr.write(`Failed getting docker image for ${nodeVersion}`)
+  process.exit(1)
+}
+process.stdout.write(version)
```

Comment:
```suggestion
console.error(version)
```

Reply:
No this is an output that should go to stdout